### PR TITLE
Fix a few things that were missed re translations and plan templates in the Admin UI and API

### PR DESCRIPTION
**Pull request by davisagli**

### CODE REVIEW

File: metadeploy/api/models.py [hunk 1]
```diff
@@ -377,6 +378,9 @@ def preflight_message_markdown(self):
     def post_install_message_markdown(self):
         return self.get_translation("en-us").post_install_message_markdown
 
+    def __str__(self):
+        return self.name
+
```
Comment (jgerigmeyer) [inline metadeploy/api/models.py]: This is a bit unfortunate when PlanTemplates don't have `name`s by default...

screenshot 2019-02-14 14 15 19

Comment (davisagli) [reply metadeploy/api/models.py]: Hmm, is that left from a migration before I asked Kit to make plan templates optional? On my local instance there's only one PlanTemplate, which I gave a name.

Comment (jgerigmeyer) [reply metadeploy/api/models.py]: Oh, I think it's from the `populate_data` script. I can make that provide a name.

Comment (davisagli) [reply metadeploy/api/models.py]: Thanks -- sorry I missed that.

**Pull request merged**